kedro-org/kedro-website
# Feature/add hero animations

## Description

Resolves #20.

## Development notes

Wow, what a pain. The animations are sorted now. They don't match exactly 100%, but they're very close. I still don't have that little rocket ship icon, as discussed in the standup today, but we can add that later.

I've further finessed the mobile menu animation as well as added in the favicon, which required adding a [custom document page](https://nextjs.org/docs/advanced-features/custom-document). That's a small change though.

Lastly, I added a `useWindowSize` hook to our `/utils/hooks/` folder. I need to be able to determine what width to use for the hero animation when we're in our mobile breakpoint, and this is how I've done it.

## QA notes

Please reference [the issue](https://github.com/kedro-org/kedro-website/issues/20) to view the specific prototypes for the animations to check against and have a look at the website. Hover over the hero buttons and open/close the mobile menu.

## Checklist
